packages/cli/commons/src/platform/authenticatedClient.ts

```diff
@@ -39,12 +41,47 @@ export class AuthenticatedClient {
     if (!Object.keys(global).includes('FormData')) {
       Object.assign(global, {FormData});
     }
+
     return new PlatformClient({
       globalRequestSettings,
       environment: castEnvironmentToPlatformClient(resolvedConfig.environment),
       region: resolvedConfig.region,
       organizationId: resolvedConfig.organization,
       accessToken: resolvedConfig.accessToken!,
+      responseHandlers: [
+        {
+          canProcess: (response: Response): boolean => {
+            return response.ok;
+          },
+          process: async <T>(response: Response) => {
+            const content = await response.text();
+            return (
+              JSON.parse as (
+                text: string,
+                reviver?: (
+                  key: string,
+                  value: any,
+                  context: {source: any}
+                ) => any | undefined
+              ) => any
+            )(content, (key, value, context) => {
+              if (
+                key === 'rowid' &&
+                typeof value === 'number' &&
+                !Number.isSafeInteger(value)
```

Comment:
I limited it to the `rowid` key because it's the only value that actually needs to be fixed at the moment (at least, as far as we know), but it's probably a good idea to do that, yes.